composer.json

```diff
@@ -16,10 +16,10 @@
     },
     "license": "GPL-3.0+",
     "require": {
-        "php": ">=5.4.0",
-        "ext-bcmath": "*"
+        "php": ">=5.4.0"
     },
     "require-dev": {
+        "ext-bcmath":                               "*",
```

Comment:
I don't understand, we talk about it on https://github.com/ronanguilloux/IsoCodes/issues/7#issuecomment-94947772.

`bcmod` is required **only** for Iban and Bban validator.

This solution premit user to run your lib without `ext-bcmath` if those validator are not required.

It's required on dev section for tests btw.